Sources/JobsPostgres/PostgresJobsQueue.swift

```diff
@@ -190,16 +190,17 @@ public final class PostgresJobQueue: JobQueueDriver {
 
                     let stream = try await connection.query(
                         """
-                        DELETE
-                        FROM _hb_pg_job_queue pse
-                        WHERE pse.job_id =
-                            (SELECT pse_inner.job_id
-                            FROM _hb_pg_job_queue pse_inner
-                            WHERE delayed_until IS NULL OR delayed_until <= Now() + INTERVAL '5 MINUTES'
-                            ORDER BY pse_inner.createdAt ASC
-                                FOR UPDATE SKIP LOCKED
-                            LIMIT 1)
-                        RETURNING pse.job_id
+                        DELETE FROM
+                            _hb_pg_job_queue
+                        USING (
+                            SELECT job_id FROM _hb_pg_job_queue
+                            WHERE (delayed_until IS NULL OR delayed_until <= NOW() AT TIME ZONE 'utc')
+                            ORDER BY createdAt, delayed_until ASC
+                            LIMIT 1
+                            FOR UPDATE SKIP LOCKED
+                        ) queued
+                        WHERE queued.job_id = _hb_pg_job_queue.job_id
+                        RETURNING _hb_pg_job_queue.job_id
```

Comment:
I changed the structure of the query because it was much easier to read (reason about). I can change it back if needed. The overall goal is to hopefully transition to just one table and allow user to pass config for data retention. (1 week, 90 day e.t.c)